docs/05-technical-reference/00-custom-resources/telemetry-01-logpipeline.md

```diff
@@ -61,67 +61,85 @@ For further LogPipeline examples, see the [samples](https://github.com/kyma-proj
-| output.http.tls | object | TLS Configuration of the HTTP target server.  |
-| output.http.tls.disabled | boolean | Indicates if TLS is disabled or enabled. Default is `false`. |
-| output.http.tls.skipCertificateValidation | boolean | If `true`, the validation of certificates is skipped. Default is `false`. |
-| output.http.uri | string | URI for the target HTTP server. Fluent Bit Default is `/`. |
-| output.http.user | object | Basic Auth username.|
-| output.http.user.value | string | Username value, can contain references to Secret values. |
-| output.http.user.valueFrom.secretKeyRef | object | Reference to a key in a Secret. You must provide `name` and `namespace` of the Secret, as well as the name of the `key`. |
-| output.custom | string | Any other Fluent Bit output specified in the Fluent Bit configuration syntax. **Note:** If you use a `custom` output, you put the LogPipeline in [unsupported mode](./../../01-overview/main-areas/telemetry/telemetry-02-logs.md#unsupported-mode#unsupported-mode). |
-| variables | []object | A list of mappings from Kubernetes Secret keys to environment variables. Mapped keys are mounted as environment variables, so that they are available as [Variables](https://docs.fluentbit.io/manual/administration/configuring-fluent-bit/classic-mode/variables) in the sections.|
-| variables[].name | string | Name of the variable to map. |
-| variables[].valueFrom.secretKeyRef | object | Reference to a key in a Secret. You must provide `name` and `namespace` of the Secret, as well as the name of the `key`.|
-| files | []object | A list of text snippets that are mounted as files to Fluent Bit, so that they are available for reference in filters and outputs. The mounted snippet is available under the `/files` folder.|
-| files[].name | string | The file name under which the snippet is mounted. The resulting path will be `/files/<name>`. |
-| files[].content | string | The actual text snippet to mount as file.|
-
-
-### status attribute
-
-For details, see the [LogPipeline specification file](https://github.com/kyma-project/telemetry-manager/blob/main/apis/telemetry/v1alpha1/logpipeline_types.go).
-
-| Parameter | Type | Description |
-|---|---|---|
-| conditions | []object | An array of conditions describing the status of the pipeline.
-| conditions[].lastTransitionTime | []object | An array of conditions describing the status of the pipeline.
-| conditions[].reason | []object | An array of conditions describing the status of the pipeline.
-| conditions[].type | enum | The possible transition types are:<br>- `Running`: The instance is ready and usable.<br>- `Pending`: The pipeline is being activated. |
-| unsupportedMode | bool | Is active when the LogPipeline uses a `custom` output or filter; see [unsupported mode](./../../01-overview/main-areas/telemetry/telemetry-02-logs.md#unsupported-mode#unsupported-mode).
+<!-- TABLE-START -->
+<!-- LogPipeline -->
```

Comment:
if easy possible, please add group and version to the string as well to have it specific. We will have multiple versions in future